.travis.yml

```diff
@@ -1,8 +1,16 @@
 language: python
+dist: bionic
 python:
 - '3.6'
```

Comment:
Weren't we going to drop 3.6?

Reply:
One of our customers is currently using 3.6 to run their ingest pipeline.  The customer team is following up to understand why that is, but so long as it is we need to support it.